src/base58.cpp

```diff
@@ -84,44 +90,45 @@ static const int8_t mapBase58[256] = {
     return true;
 }
 
+auto ConvertToLongs(const Span<const unsigned char>& input, size_t leadingZeros) -> std::vector<int64_t> {
```

Comment:
converting from base(256) to base(256^7) speeds up processing since we can do actual division and modulo on the resulting int64